azure-steps-windows.yml

```diff
@@ -32,15 +40,15 @@ steps:
         refreshenv
     }
     python -c "from tools import openblas_support; openblas_support.make_init('numpy')"
-    pip wheel -v -v -v --wheel-dir=dist .
+    pip wheel -v -v -v --no-build-isolation --no-use-pep517 --wheel-dir=dist .
```

Comment:
While not really part of the openblas handling, this will disable all the fancy "copy to tmp directory, exec(setup.py)" things that pip now does when calling `pip wheel`. The former slows things down, the latter breaks our setup.py (it cannot find `parse_setuppy_commands`)